cmd/earth/main.go

```diff
@@ -1480,7 +1543,8 @@ func (app *earthApp) collectAnalytics(exitCode int, realtime time.Duration) {
 			Set("version", Version).
 			Set("gitsha", GitSha).
 			Set("exitcode", exitCode).
-			Set("ci", ci).
+			Set("ci", detectCI()).
+			Set("repo", getRepoHash()).
```

Comment:
`repohash` for clarity?

```suggestion
			Set("repohash", getRepoHash()).
```